### PR TITLE
Stuff

### PR DESCRIPTION
**Pull request by antonbobkov**

### CODE REVIEW

File: erlang/potatoserver/test/my_crypto_test.erl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 -module(my_crypto_test).
 
--export([my_crypto_test/0]).
+-include_lib("eunit/include/eunit.hrl").
```
Comment (pdlla) [inline erlang/potatoserver/test/my_crypto_test.erl]: oh I didn't realize we needed to do this ʘ‿ʘ
does that mean none of the unit tests were running before?

**Comment by pdlla** [comment]
ʕ •`ᴥ•´ʔ

File: erlang/potatoserver/test/my_crypto_test.erl (hunk 1 above)
Comment (antonbobkov) [reply erlang/potatoserver/test/my_crypto_test.erl]: we were good before!

this just auto exports all functions ending with "_test"

Comment (pdlla) [reply erlang/potatoserver/test/my_crypto_test.erl]: such actually reading through eunit documentation

**Pull request merged**